content/en/docs/components/training/user-guides/managedby.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "spec.managedBy"
+Desciption = "Using managedBy feild for MultiKueue"
+weight = 60
++++
+
+## Overview
+
+The `spec.managedBy` field is a new feature introduced for MultiQueue support in the Kubeflow Training Operator. This field allows for more robust management of multi-cluster job dispatching by specifying the managing entity.
+
+## Prerequisites
+
+1. Ensure that you have the latest version of the Kubeflow Training Operator installed.
+2. Make sure Kueue is compiled against the new operator to leverage the `spec.managedBy` field.
+
+## Usage
+
+To use the `spec.managedBy` field in your training jobs, include it in the job specification as shown below:
+
+```yaml
+apiVersion: "kubeflow.org/v1"
+kind: "TFJob"
+metadata:
+ name: "example-tfjob"
+spec:
+ managedBy: "kueue"
+ tfReplicaSpecs:
+   ...
+```
+
+Example
```

Comment:
Actually, underneath Kueue will add a webhook which will default the field, so the example will not need to include it. I think we can just rely on the example eventually added to the MultiKueue documentation once https://github.com/kubernetes-sigs/kueue/issues/2552 is done.

Reply:
WDYT @tenzen-y  @andreyvelich ?

Reply:
Yes, I think we can reference to the example in Kueue docs, so we can be consistent.

Reply:
ok, I suggest to drop it in this PR so that we only have one place about the details. Note that the Kueue documentation is not added yet. Once done we will x-ref the pages.